seedsource_core/django/seedsource/views.py

```diff
@@ -22,14 +22,15 @@
 from .utils import get_elevation_at_point, get_regions_for_point
 
 SEEDSOURCE_TITLE = getattr(settings, 'SEEDSOURCE_TITLE', 'seedsource-core')
-
+MBTILESERVER_ROOT = getattr(settings, 'MBTILESERVER_ROOT', 'seedsource-core')
```

Comment:
Please give this a better default (`seedsource-core` is probably _never_ going to work in any environment). E.g: `/`.